fdbclient/include/fdbclient/StorageServerInterface.h

```diff
@@ -1266,6 +1270,51 @@ struct GetHotShardsRequest {
 	}
 };
 
+struct CheckSumMetaData {
+	constexpr static FileIdentifier file_identifier = 3828142;
+	KeyRange range;
+	Version version;
+	StringRef checkSumValue;
+
+	CheckSumMetaData() {}
+	explicit CheckSumMetaData(KeyRange range, Version version, StringRef checkSumValue)
```

Comment:
nit: explicit is unnecessary here. https://en.cppreference.com/w/cpp/language/explicit